integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java

```diff
@@ -223,11 +237,19 @@ void testWlsRemoteConsoleConnectionUsingSSL() {
     logger.info("WebLogic console is accessible thru default-secure service");
 
     //verify remote console is accessible through default-secure nodeport
-    curlCmd = "curl -sk -v --show-error --noproxy '*' --user weblogic:welcome1 -H "
+    /*curlCmd = "curl -sk -v --show-error --noproxy '*' --user weblogic:welcome1 -H "
         + "Content-Type:application/json -d "
         + "\"{ \\" + "\"domainUrl\\" + "\"" + ": " + "\\" + "\"" + "https://"
         + hostAndPort + "\\" + "\" }" + "\""
-        + " http://localhost:8012/api/connection  --write-out %{http_code} -o /dev/null";
+        + " http://localhost:8012/api/connection  --write-out %{http_code} -o /dev/null";*/
+    curlCmd = "curl -sk -v --show-error --noproxy '*' --user weblogic:welcome1 "
```

Comment:
Can u make it as function and call with a parameter with treaefik/nginx port

Reply:
Will do.